other/record-creation-details/.chainsaw-test/chainsaw-test.yaml

```diff
@@ -5,6 +5,11 @@ metadata:
   creationTimestamp: null
   name: record-creation-details
 spec:
+  bindings:
+  - name: version
+    value: ($kubeVersion)
```

Comment:
this should be `$values.kubeVersion` (values are available under `$values` binding, not at the root)

Reply:
I already fixed it.